modules/aruco/samples/aruco_dict_utils.cpp

```diff
@@ -0,0 +1,276 @@
+#include <opencv2/core/hal/hal.hpp>
+#include <opencv2/aruco.hpp>
+#include <iostream>
+
+using namespace cv;
+using namespace std;
+
+static inline Mat _generateRandomMarker(int markerSize, RNG &rng) {
+    Mat marker(markerSize, markerSize, CV_8UC1, Scalar::all(0));
+    for(int i = 0; i < markerSize; i++) {
+        for(int j = 0; j < markerSize; j++) {
+            unsigned char bit = (unsigned char) (rng.uniform(0,2));
+            marker.at< unsigned char >(i, j) = bit;
+        }
+    }
+    return marker;
+}
```

Comment:
You can use RNG::fill method instead of hand-made function: https://docs.opencv.org/4.x/d1/dd6/classcv_1_1RNG.html#ad26f2b09d9868cf108e84c9814aa682d

Reply:
fixed

Reply:
`_generateRandomMarker()` was removed